frontends/java/soot/src/main/java/ossf/fuzz/introspector/soot/CallGraphGenerator.java

```diff
@@ -280,16 +283,24 @@ protected void internalTransform(String phaseName, Map<String, String> options)
 				classConfig.setFunctionConfig(methodConfig);
 				classYaml.add(classConfig);
 		}
-		String callTree = extractCallTree(callGraph, this.entryMethod, 0, -1);
-		System.out.println(callTree);
-		System.out.println("--------------------------------------------------");
-		ObjectMapper om = new ObjectMapper(new YAMLFactory());
-		for(FuzzerConfig config:classYaml) {
-			try {
-				System.out.println(om.writeValueAsString(config) + "\n");
-			} catch (JsonProcessingException e) {
-				e.printStackTrace();
-			}
+		try {
+			File file = new File("fuzzerLogFile-" + this.entryClassStr + ".data");
+			file.createNewFile();
+			FileWriter fw = new FileWriter(file);
+			fw.write(extractCallTree(callGraph, this.entryMethod, 0, -1));
+		    fw.close();
```

Comment:
some formatting/indentation issues? as this code matures we should start to adopt some more stringent linting -- feel free to add some linting checks in the CI as well when you're ready.